server/sandbox_run.go

```diff
@@ -22,6 +22,12 @@ import (
 	pb "k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/runtime"
 )
 
+const (
+	// TODO: Remove this const once this value is provided over CRI
+	// See https://github.com/kubernetes/kubernetes/issues/47938
```

Comment:
@TomSweeneyRedHat I am following the upstream issue and will make the change here once it is resolved.